### PR TITLE
[DGS-422]: handle skipped steps

### PR DESCRIPTION
**Pull request by wolfderechter**
## ID
 DGS-422

 ## Description

Handle skipped steps (as seen in LEKP 1.0 subsidies). So we don't have an error when opening steps that haven't been created.

 ## Type of change

 - [ ] Bug fix
 - [x] New feature
 - [ ] Breaking change
 - [ ] Other

 ## How to test

Open LEKP 1.0 subsidies, and check the different steps.

 ## Links to other PR's

 - https://github.com/lblod/frontend-subsidiepunt/pull/28

### CODE REVIEW

File: app/templates/subsidy/detail/step/skipped.hbs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div id="subsidyParent" class="au-c-body-container au-c-body-container--scroll">
+  <div class="au-o-box au-o-box--small">
+    <div class="au-c-alert au-c-alert--warning">
```
Comment (Windvis) [inline app/templates/subsidy/detail/step/skipped.hbs:3]: Why not use the AuAlert component?

File: app/templates/subsidy/detail/step/detail.hbs [hunk 2]
```diff
@@ -180,37 +133,69 @@
...
+                      <a
+                        class="au-c-link"
+                        href="mailto:LoketLokaalBestuur@vlaanderen.be"
+                      >
+                        LoketLokaalBestuur@vlaanderen.be
+                      </a>.
+                    </p>
+                  </AuContent>
                 </AuAlert>
+                {{! Buttons not needed in readonly detail view }}
```
Comment (Windvis) [inline app/templates/subsidy/detail/step/detail.hbs:166]: Why not simply remove this instead of commenting it?

Comment (wolfderechter) [reply app/templates/subsidy/detail/step/detail.hbs:166]: Removed it now, we wanted to keep it around for a little bit in case we ended up needing it.

**Review by aatauil — APPROVED** [review]
LGTM

**Pull request merged**